### PR TITLE
Choose Color before starting computer Game

### PR DESCRIPTION
**Pull request by SamGonChec**
# Description

Hey guys, so I was able to implement a choose color before a computer game in our program. What I implemented was two separate methods in the gamemanager class, and a conditional in the tutorial class. Finally I fixed a small error I made in the last push I made to the repo. My bad.

I have removed unnecessary lines of code in all of our files. Specially in the include header files. Lastly, I have detached the tutorial screen from appearing at the second player method. Now you can go to the tutorial then back to the main menu if you wish.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Thumbs up on the request message I posted on discord if you are reading this.
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by rpwgc4 — APPROVED** [review]
I'm approving but it would be nice if you could go over your code with an eye for style. There's a few conspicuous missing spaces. Also, the Qt style guidelines we're using say that an else statement should be on the same line as the preceding curly brace like this:

`} else {`

Weird, I know.

File: NineMensMorris/src/singleplayerscreen.cpp
Comment (mumkymikey) [inline NineMensMorris/src/singleplayerscreen.cpp]:
```suggestion
    QGraphicsTextItem *chooseColor = scene->addText("Choose your color");
```
I think making the prompt simpler like above would help with readability

**Review by anodyne99 — APPROVED** [review]
I'll have to test the splash screen again after you merge this.

**Review by jvnyxc — APPROVED** [review]

**Review by rpwgc4 — APPROVED** [review]

**Review by mumkymikey — APPROVED** [review]

**Pull request merged**
